cmd/utils/flags.go

```diff
@@ -271,20 +270,20 @@ var (
 	WSListenAddrFlag = cli.StringFlag{
 		Name:  "wsaddr",
 		Usage: "WS-RPC server listening interface",
-		Value: "127.0.0.1",
+		Value: common.DefaultWsHost,
 	}
 	WSPortFlag = cli.IntFlag{
 		Name:  "wsport",
 		Usage: "WS-RPC server listening port",
-		Value: 8546,
+		Value: common.DefaultWsPort,
 	}
 	WSApiFlag = cli.StringFlag{
 		Name:  "wsapi",
 		Usage: "API's offered over the WS-RPC interface",
 		Value: rpc.DefaultHttpRpcApis,
 	}
-	WSAllowedDomainsFlag = cli.StringFlag{
-		Name:  "wsdomains",
+	WSCORSDomainFlag = cli.StringFlag{
```

Comment:
The websocket standard doesn't support CORS as the HTTP standard does. Therefore the current implementation gives the user the option to specify a list of domains from which websockets are allowed. This check is server side (HTTP CORS is enforced by the browser). Therefore both are not the same and I would recommend leaving the old name in place.